DashKit/PumpManagerAlert.swift

```diff
@@ -185,7 +189,10 @@ extension PumpManagerAlert: RawRepresentable {
         case "multiCommand":
             self = .multiCommand
         case "userPodExpiration":
-            self = .userPodExpiration
+            guard let offset = rawValue["offset"] as? TimeInterval else {
+                return nil
```

Comment:
[nit] Will this cause backward-compatible issues? Could you use a default offset if the `rawValue["offset"]` is not there (e.g. `PodExpirationAlert.defaultTimeIntervalAlert`)?

Reply:
Yeah, if we had a previous version to support, I'd want to version this. For active testers, I don't think we'll see any impacts; you'd have to wait for the user expiration alert on a previous version, ignore the alert, kill  the app, install the new version, and restart the app, and then acknowledge the alert, and in that case what would happen is the pod alert would still be active on the pod, and wouldn't be cleared properly. We can already get into this state if comms fail during the alert ack (which is tracked by a separate ticket).